hotshot-state-prover/src/service.rs

```diff
@@ -645,11 +646,13 @@ mod test {
     async fn test_read_contract_state() -> Result<()> {
         setup_logging();
         setup_backtrace();
-
+        println!("deploy contract");
```

Comment:
change to log?

Reply:
ahh this was during debugging at one point, will remove

Reply:
https://github.com/EspressoSystems/espresso-sequencer/commit/12b6bf256ec4136d6a2d62edaf40809818d7e6d8#diff-d85261c27bf48fe0ec00e8cc9bb3ca101d708ec68c85968531b2e2cf4f9ed292